<ui version="4.0" >
 <class>dialogsliceClass</class>
 <widget class="QWidget" name="dialogsliceClass" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>5</width>
    <height>2</height>
   </size>
  </property>
  <property name="mouseTracking" >
   <bool>false</bool>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Number of planes</string>
        </property>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QSpinBox" name="spinBox" >
           <property name="buttonSymbols" >
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Distance between planes</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QToolButton" name="DefultButton" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Restore Default</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DistanceEdit" >
             <property name="text" >
              <string>1</string>
             </property>
             <property name="echoMode" >
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="cursorPosition" >
              <number>1</number>
             </property>
             <property name="alignment" >
              <set>Qt::AlignHCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelDistanceRange" />
         </item>
         <item>
          <widget class="QSlider" name="SliderPlaneDistance" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor" >
            <cursorShape>SizeHorCursor</cursorShape>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="value" >
            <number>1</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedControls" >
            <bool>true</bool>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval" >
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QGroupBox" name="groupBox_4" >
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="Exporter_singleFile" >
             <property name="text" >
              <string>Single file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Exporter_MultiFile" >
             <property name="text" >
              <string>Multi file</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>121</width>
            <height>68</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="ExportButton" >
          <property name="text" >
           <string>Export Polyline...</string>
          </property>
          <property name="popupMode" >
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>SliderPlaneDistance</tabstop>
  <tabstop>DefultButton</tabstop>
  <tabstop>ExportButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
